src/Compilers/CSharp/Test/Semantic/Semantics/LocalFunctionTests.cs

```diff
@@ -445,12 +442,73 @@ void M()
             Assert.NotNull(symbol);
 
             var attributes = symbol.GetAttributes();
-            // PROTOTYPE: method symbol should have the attribute
-            //Assert.Single(attributes);
+            Assert.Single(attributes);
+
+            var attributeData = attributes[0];
```

Comment:
Minor point: these could be combined in each of these tests.
`var attributeData = attributes.Single();`